cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1406,21 +1406,17 @@ if(vc AND NOT Vc_FOUND AND NO_CONNECTION)
 endif()
 
 if(vc AND NOT Vc_FOUND)
-  set(Vc_VERSION "1.4.1")
+  set(Vc_VERSION "1.4.3")
   set(Vc_PROJECT "Vc-${Vc_VERSION}")
   set(Vc_SRC_URI "${lcgpackages}/${Vc_PROJECT}.tar.gz")
   set(Vc_DESTDIR "${CMAKE_BINARY_DIR}/externals")
   set(Vc_ROOTDIR "${Vc_DESTDIR}/${CMAKE_INSTALL_PREFIX}")
   set(Vc_LIBNAME "${CMAKE_STATIC_LIBRARY_PREFIX}Vc${CMAKE_STATIC_LIBRARY_SUFFIX}")
   set(Vc_LIBRARY "${Vc_ROOTDIR}/lib/${Vc_LIBNAME}")
 
-  if(UNIX)
-    set(VC_PATCH_COMMAND patch -p1 < ${CMAKE_SOURCE_DIR}/cmake/patches/vc-deprecated-and-bit-scan-forward.patch)
-  endif()
-
   ExternalProject_Add(VC
     URL     ${Vc_SRC_URI}
-    URL_HASH SHA256=68e609a735326dc3625e98bd85258e1329fb2a26ce17f32c432723b750a4119f
+    URL_HASH SHA256=988ea0053f3fbf17544ca776a2749c097b3139089408b0286fa4e9e8513e037f
     PATCH_COMMAND ${VC_PATCH_COMMAND}
```

Comment:
You need to drop this command and associated patch, they are in Vc 1.4.2 already, so the patch is failing on 1.4.3.

Reply:
Yeah totally did this in my second attempt :-)

Reply:
Yes, not the `${VC_PATCH_COMMAND}` is empty, but you don't need the line
```
PATCH_COMMAND ${VC_PATCH_COMMAND}
```
anymore, that's what I meant. You didn't remove this line in your second attempt.

Reply:
Thanks, fixed!